src/awkward/numba.py

```diff
@@ -33,6 +37,119 @@ def register_and_check():
     _register()
 
 
+class GrowableBuffer:
+    def __init__(self, dtype, *, initial=1024, resize=8.0):
+        register_and_check()
+
+        import numba
```

Comment:
I have verified that this `GrowableBuffer` class is exactly the one removed from src/awkward/_connect/numba/growablebuffer.py with this

```python
        register_and_check()
￼
￼        import numba
```

added. (This is a note for posterity, since git isn't able to tell us that this is a move, rather than a random addition and deletion.)